packages/openai-compatible/src/openai-compatible-embedding-model.test.ts

```diff
@@ -97,9 +97,14 @@ describe('doEmbed', () => {
   it('should pass the dimensions setting', async () => {
     prepareJsonResponse();
 
-    await provider
-      .textEmbeddingModel('text-embedding-3-large', { dimensions: 64 })
-      .doEmbed({ values: testValues });
+    await provider.textEmbeddingModel('text-embedding-3-large').doEmbed({
+      values: testValues,
+      providerOptions: {
+        'openai-compatible': {
+          dimensions: 64,
+        },
+      },
+    });
 
```

Comment:
embedding models do not support provider options

Reply:
resolving as we now added